contracts/BSCValidatorSet.sol

```diff
@@ -491,7 +493,7 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
       || (maxNumOfMaintaining == 0 || maintainSlashScale == 0)      // - 1. check if not start
       || numOfMaintaining >= maxNumOfMaintaining                    // - 2. check if reached upper limit
       || !isWorkingValidator(index)                                 // - 3. check if not working(not jailed and not maintaining)
-      || validatorExtraSet[index].enterMaintenanceHeight > 0        // - 5. check if has Maintained
+      || validatorExtraSet[index].enterMaintenanceHeight > 0        // - 5. check if has Maintained during one 24-hour period
```

Comment:
I am confused by `during one 24-hour period`

Reply:
`EnterMaintenanceHeight` is cleared to zero in each 24-hour period,  `validatorExtraSet[index].enterMaintenanceHeight > 0` could be only used to check if has Maintained in one 24-hour period,  this is a more accurate description